boards/arm/pan1781_evb/pan1781_evb.dts

```diff
@@ -0,0 +1,150 @@
+/*
+ * PAN1781 EVB configuration
+ *
+ * Copyright (c) 2022 Panasonic Industrial Devices Europe GmbH
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <nordic/nrf52820_qdaa.dtsi>
+#include "pan1781_evb-pinctrl.dtsi"
+
+/ {
+	model = "pan1781-evb";
+	compatible = "panasonic-industrial-devices-europe-gmbh,pan1781-evb";
+
+	chosen {
+		zephyr,console = &uart0;
+		zephyr,shell-uart = &uart0;
+		zephyr,uart-mcumgr = &uart0;
+		zephyr,bt-mon-uart = &uart0;
+		zephyr,bt-c2h-uart = &uart0;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+		zephyr,code-partition = &slot0_partition;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		evb_led1: evb_led_0 {
+			gpios = <&gpio0 14 GPIO_ACTIVE_LOW>;
+			label = "LED1 on EVB";
+		};
+		evb_led2: evb_led_2 {
+			gpios = <&gpio0 15 GPIO_ACTIVE_LOW>;
+			label = "LED2 on EVB";
+		};
+	};
+
+	pwmleds {
+		compatible = "pwm-leds";
+		pwm_evb_led1: pwm_evb_led_1 {
+			pwms = <&sw_pwm 0 PWM_MSEC(20) PWM_POLARITY_INVERTED>;
+		};
+	};
```

Comment:
where is `sw_pwm` enabled?

Reply:
Yes you are right, i can see that `sw_pwm` node in `nrf52820.dtsi` is disabled by default, and it will not be enabled on another file. But our template for generation of this devicetree was the file `nRF52833dk_nrf52820.dts` from Nordic. Here it was done exactly the same way, we have done. What would you suggest? Do i have to add the `status = "okay";` to the node, before i get an go from you?

Reply:
yes, and you also need to setup to which GPIO signal needs to be generated, see e.g. `boards/arm/nrf51dk_nrf51422/nrf51dk_nrf51422.dt`

Reply:
Ah okay. Thanks a lot for this hint to `boards/arm/nrf51dk_nrf51422/nrf51dk_nrf51422.dts`. I have updated the device tree like you've suggested.